src/libraries/Microsoft.PowerFx.Interpreter/EvalVisitor.cs

```diff
@@ -179,7 +181,7 @@ public override async ValueTask<FormulaValue> Visit(CallNode node, SymbolContext
             {
                 if (FuncsByName.TryGetValue(func, out var ptr))
                 {
-                    var result = await ptr(this, childContext, node.IRContext, args);
+                    var result = await ptr(this, new EvalVisitorContext(childContext, context.StackMarker), node.IRContext, args);
```

Comment:
>StackMarker [](http://example.com/codeflow?start=94&length=11)

Can this be:
context.StackMarker.Inc() 


And then remove this above:
  context.StackMarker = context.StackMarker.Inc();

Reply:
Changed to a EvalVisitorContext#IncrementStackDepthCounter